### ISSUE TITLE
biomarkt null id

### ISSUE THREAD

**Issue by TimoKramer** [yobst/heisse-preise-de#34]
when starting the service today I had an exception of missing id and failed input into the db:
![image](https://github.com/yobst/heisse-preise-de/assets/4785848/1498c500-9a0c-4ff9-ac29-cf032ab45818)
```
Error inserting data: error: null value in column "id" of relation "prices" violates not-null constraint
    at /home/timo/projects/yobst-scraper/server/build/server.js:52:11175
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async _y (/home/timo/projects/yobst-scraper/server/build/server.js:76:3074) {
  length: 316,
  severity: 'ERROR',
  code: '23502',
  detail: 'Failing row contains (biomarkt, null, Clementinen, All, false, $2.85, [{"date": "2024-01-09", "price": 2.85, "unitPrice": 0}], f, g, 1000, f, null, , f, f).',
  hint: undefined,
  position: undefined,
  internalPosition: undefined,
  internalQuery: undefined,
  where: undefined,
  schema: 'public',
  table: 'prices',
  column: 'id',
  dataType: undefined,
  constraint: undefined,
  file: 'execMain.c',
  line: '2003',
  routine: 'ExecConstraints'
}
```

**Comment by TimoKramer** [comment]
the image is my output for debugging

**Comment by jsmassa** [comment]
Solved with PR #38
